rust/platform/cpu/BUILD

```diff
@@ -0,0 +1,6 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
nit: can you default to private and set public explicitly?

Reply:
Done.